src/main/java/Ava.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
+
+public class Ava {
+
+    public static void main(String[] args) {
+        greet();
+        mainProcess();
+        exit();
+    }
+
+    public static void mainProcess() {
+        boolean isExit = false;
+        Task[] tasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        while (!isExit) {
+            String task = in.nextLine();
+            if (task.equals("bye")) {
+                isExit = true;
+                continue;
+            } else if (task.equals("list")) {
+                listTask(tasks);
+                continue;
+            } else if (task.contains("mark")) {
+                markTask(tasks, task);
+                continue;
+            } else if (task.startsWith("todo")) {
+                task = task.replace("todo ", "");
+                tasks[Task.getNumberOfTasks()] = new ToDo(task);
+            } else if (task.startsWith("deadline")) {
+                String[] taskAndDate = trimTask(task, "deadline ");
+                tasks[Task.getNumberOfTasks()] = new Deadline(taskAndDate[0], taskAndDate[1]);
+            } else if (task.startsWith("event")) {
+                String[] taskAndDate = trimTask(task, "event ");
+                tasks[Task.getNumberOfTasks()] = new Event(taskAndDate[0], taskAndDate[1], taskAndDate[2]);
```

Comment:
I noticed that there are different levels of abstraction between these else-if blocks. Perhaps you could consider moving the body under the `task.startsWith("todo")`, `task.startsWith("deadline")` and `task.startsWith("event")` blocks to separate methods to enhance readability?